{
  "current_language": "Русский",
  "current_language_short": "ru",
  "login": "Вход",
  "email": "Почта",
  "password": "Пароль",
  "firstName": "Имя",
  "lastName": "Фамилия",
  "continue_with_github": "Продолжить с GitHub",
  "or": "или",
  "sign_in_heading": "Вход",
  "sign_in_action_text": "Войти",
  "sign_up_heading": "Создание аккаунта",
  "sign_up_action_text": "Создать",
  "no_account_question": "Нет аккаунта?",
  "already_have_account_question": "Уже есть аккаунт?",
  "error": {
    "general": {
      "unknown_error": "Что-то пошло не так. Пожалуйста, попробуйте еще раз."
    },
    "api": {
      "forbidden_error": "Нет доступа",
      "auth_error": "Ошибка авторизации",
      "validation_failed": "Ошибка валидации",
      "email_already_used": "Этот email уже занят",
      "invalid_credentials": "Недействительные данные",
      "github_connected_to_current_user": "Этот GitHub-аккаунт уже привязан к вашему аккаунту.",
      "github_connected_to_another_user": "Этот GitHub-аккаунт уже привязан к другому пользователю."
    },
    "validation": {
      "unknown_issue": "Неизвестная ошибка валидации",
      "invalid_string": "Поле $t({{field}}) введено некорректно",
      "too_small": "Поле $t({{field}}) слишком короткое. Минимальная длина: {{min}}",
      "too_big": "Поле $t({{field}}) слишком длинное. Максимальная длина: {{max}}",
      "invalid_type": "Поле $t({{field}}) не может быть пустым"
    }
  }
}
